broadway/api/daos/grading_run.py

```diff
@@ -10,6 +10,7 @@
 class GradingRunDao(BaseDao):
     ID = "_id"
     STATE = "state"
+    COURSE_ID = "course_id"
```

Comment:
So I am not sure if we should do this. This adds some redundancy to the data structure. As in, the grading run is an instance of an assignment. And an assignment is what belongs to a course. The assignment structure already contains the `course_id`. I have commented below how you can access it.

The problem with adding redundancy is: the course id and assignment id can go out of sync (assignment id could point to an assignment which has a diff course id). This is generally considered bad design. Redundancy should be added in situations where it really boosts performance and performance is important.

Reply:
Fixed. Didn't realize that the course_id was already in the assignment struct.